docs/introduction_tutorial/intro_tutorial_part1.ipynb

```diff
@@ -102,9 +102,9 @@
     "### Create a URL Template String\n",
```

Comment:
<p>I would suggest we keep these as they were before. The idea here is to consider conceptually how we are modifying the strings, not get bogged down on having to read the strftime sytnax yet.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pangeo-forge/pangeo-forge-recipes/pull/343/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Reply:
<p>I don't quite understand what you mean by "keep these". Do you mean revert the text back to the original here? I was changing it because I am suggesting to use direct string interpolation here (rather than have placeholders that expect variables with names <code>{yyyymm}</code> for example). I also think having a variable called <code>yyyymm</code> in the string is a bit confusing because then the reader might think that the way to achieve the <code>%Y%m</code> format is to call your variable <code>yyyymm</code>.</p><p> </p><p>Maybe I could just change the text to </p><p> </p><blockquote>where <code>{time:%Y%m}</code> and <code>{time:%Y%m%d}</code> will change for each file by changing the value of <code>time</code>. Of the three ...</blockquote>

Reply:
To me, the main learning goal at this early stage in the tutorial is not about Python, it conceptual. I think we want to make sure the learner understands that the fundamental premise of pangeo forge recipes is that we are accessing predictably named urls that are organized according to the structure of the dataset. We aren't teaching code at this point in the tutorial, so my original comment was intended to say that we start using direct string interpolation when we actually start coding in the Building a FilePatttern section.

But that aside, I can see how it would be better to be consistent in how we write the template string. I like the clarifying text that you put in and I added a few more quick words that I hope will keep people on a conceptual level at this point in the tutorial. So this was all a looong way of saying I think it looks good!

Thanks again for contributing this!